pkg/gofr/datasource/sql/sql.go

```diff
@@ -159,18 +160,24 @@ func getDBConfig(configs config.Config) *DBConfig {
 		MaxIdleConn: maxIdleConn,
 		// only for postgres
 		SSLMode: configs.GetOrDefault("DB_SSL_MODE", "disable"),
+		Charset: configs.Get("DB_CHARSET"),
 	}
 }
 
 func getDBConnectionString(dbConfig *DBConfig) (string, error) {
 	switch dbConfig.Dialect {
 	case "mysql":
-		return fmt.Sprintf("%s:%s@tcp(%s:%s)/%s?charset=utf8&parseTime=True&loc=Local&interpolateParams=true",
```

Comment:
Also, from my perspective we could consider adding also a constant for being to configure the client `loc`

So it could also be set to UTC or anything needed

I had this exact need for an old database.

This is also a suggestion that could be addressed in another PR

Cc @aryanmehrotra @srijan-27 @vipul-rawat @Umang01-hash

Reply:
Here is the issue for both of your suggestions:

https://github.com/gofr-dev/gofr/issues/1174